src/pps/pps.proto

```diff
@@ -464,7 +464,9 @@ message PipelineInfo {
     Determined determined = 36;
   }
   Details details = 12;
-  string details_json = 13;
+  reserved "details_json";
```

Comment:
Same question about deprecation here. This one actually seems like an issue since we are renaming the field after it was released in v2.7.0?

Reply:
Renaming is generally safe because proto fields are identified by numbers. It _is_ implicated JSON or other text-formatted messages are involved, but `PipelineInfo` is not used as a text-formatted message anywhere.